### PR TITLE
Update Link API

### PR DESCRIPTION
**Pull request by alwynmathew**
- Now link API can be linked to multiple nodes at once
- Made linking as-performed nodes optional when creating them

### CODE REVIEW

File: dtp_apis/create_DTP_API.py [hunk 1]
```diff
@@ -245,28 +244,30 @@ def create_action_node(self, task_type, action_node_iri, task_iri, target_as_bui
...
-                    }
-                ]
-            }
-        ])
+        query_dict = {
+            "_classes": [self.DTP_CONFIG.get_ontology_uri('asPerformedAction')],
+            "_domain": self.DTP_CONFIG.get_domain(),
+            "_iri": action_node_iri,
+            "_visibility": 0,
+            self.DTP_CONFIG.get_ontology_uri('hasTaskType'): task_type,
```
Comment (jschlenger) [inline dtp_apis/create_DTP_API.py:252]: shouldn't the hasTaskType be part of the _outE attribute?
I'm not completely sure how exactly you structured everything but it looks like this is still the old version of the hasTaskType which Thomas told us is wrong

**Review by jschlenger — COMMENTED** [review]
Only one comment that needs to be clarified before merging

File: dtp_apis/create_DTP_API.py (hunk 1 above)
Comment (alwynmathew) [reply dtp_apis/create_DTP_API.py:252]: I see `hasTaskType` in both activity and operation nodes. These types should come from ontology, right? But these types are not added in the `DTP_config` file yet.

Comment (alwynmathew) [reply dtp_apis/create_DTP_API.py]: Fixed in https://github.com/BIM2TWIN-Team/DTP_API/pull/30/commits/f1592d3de790b597f89458e3d5edd58fc2835e22

**Review by jschlenger — APPROVED** [review]
Looks fine to me

**Pull request merged**